File: src/main/java/com/ddc/yieldster/jsonbuilder/model/MoveStep.java
package com.ddc.yieldster.jsonbuilder.model;

import lombok.Builder;
import lombok.Data;
import lombok.Value;

@Builder
@Value
@Data
public class MoveStep extends Step {
	 private String fromAsset;
	 private String toAsset;
	 private Double amount;
}

File: Yieldster-SDK/Models/MoveStep.py
from Models.Step import Step

class MoveStep(Step):
    def __init__(self):
        self.amount = None
        self.fromAsset = None
        self.toAsset = None
        self.type = "MoveStep"

    # def add_from_asset(self, from_asset):
    #     self.fromAsset = from_asset
    #
    # def add_to_asset(self, to_asset):
    #     self.toAsset = to_asset
    #
    # def amount(self, amount):
    #     self.amount = amount
